src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The excavator started working.");

```

Comment:
The message printed should be more informative and indicate that it is specifically an Excavator that started working.